src/ie_pandas.egg-info/PKG-INFO

```diff
@@ -0,0 +1,9 @@
+Metadata-Version: 2.1
```

Comment:
Be careful! `*.egg-info` should not be in git, because it's autogenerated. I recommend you to put a `.gitignore` file from https://www.gitignore.io/ and remove these files.

Reply:
Same goes with all `*.pyc` files.